p4src/linux_networking/README_LINUX_NETWORKING.md

```diff
@@ -37,17 +37,17 @@ This README describes a step by step procedure to run Linux networking scenario
 * Notes about topology:
   * VHOST ports, TAP ports, Physical LINK ports are created by GNMI CLI and LINK port is binded to the DPDK target
   * VLAN 1, VLAN 2, .... VLAN N created using Linux commands and are on top a TAP port. These VLAN ports should be equal to number of VM's that are spawned.
-  * br-int, VxLAN0 ports are created using ovs-vsctl command provided by P4-OVS and all the VLAN ports are attached to br-int using ovs-vsctl command.
+  * br-int, VxLAN0 ports are created using ovs-vsctl command provided by networking recipe and all the VLAN ports are attached to br-int using ovs-vsctl command.
```

Comment:
"Networking recipe", unlike "P4-OVS", takes a definite article ("the networking recipe")

Reply:
Done